### PR TITLE
Meenu/fix: typescript error fix

### PR DESCRIPTION
**Pull request by meenakshi-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/partners/payment-agent/_faq-data.tsx [hunk 1]
```diff
@@ -8,50 +8,20 @@ const General = () => {
...
         <>
-            <Header
-                as="p"
-                type="paragraph-1"
-                mt="16px"
-                tabletL={{
-                    type: 'paragraph-2',
-                }}
-            >
+            <Header as="p" type="paragraph-1" mt="16px">
```
Comment (konstantinosG-deriv) [inline src/pages/partners/payment-agent/_faq-data.tsx:11]: Don't we need the tabletL prop here? Maybe the `type: 'paragraph-2' ` can be given a different type to fix the error

File: src/components/elements/timeline.tsx [hunk 2]
```diff
@@ -71,16 +71,17 @@ const Oval = () => {
...
+    children?: ReactNode
 }
 
-type TimelineProps = Pick<TimelineTickProps, 'pb' | 'children'>
+type TimelineProps = Pick<TimelineTickProps, 'pb' | 'pl' | 'children'>
 
 type ItemProps = Pick<TimelineTickProps, 'children'>
 
-const Timeline = ({ pb, children, ...props }: TimelineProps) => {
+const Timeline = ({ pb, pl, children, ...props }: TimelineProps) => {
```
Comment (niloo-fs) [inline src/components/elements/timeline.tsx:84]:
```suggestion
const Timeline = ({ pb, pl, children, ...props }: React.PropsWithChildren<TimelineProps>) => {
```

File: src/components/elements/timeline.tsx [hunk 3]
```diff
@@ -71,16 +71,17 @@ const Oval = () => {
 
 type TimelineTickProps = {
     pb?: string
+    pl?: string
     is_border?: boolean
     color?: string
-    children?: ReactNode[]
+    children?: ReactNode
```
Comment (niloo-fs) [inline src/components/elements/timeline.tsx:77]:
```suggestion
    
```

File: src/pages/partners/payment-agent/_faq-data.tsx [hunk 4]
```diff
@@ -8,50 +8,20 @@ const General = () => {
...
-            <Header
-                as="p"
-                type="paragraph-1"
-                mt="8px"
-                weight="normal"
-                tabletL={{
-                    type: 'paragraph-2',
-                }}
-            >
+            <Header as="p" type="paragraph-1" mt="8px" weight="normal">
```
Comment (niloo-fs) [inline src/pages/partners/payment-agent/_faq-data.tsx:14]:
```suggestion
            <StyledHeader as="p" mt="8px" weight="normal">
```

Same for the rest of the Headers

File: src/pages/partners/payment-agent/_faq-data.tsx [hunk 5]
```diff
@@ -8,50 +8,20 @@ const General = () => {
     const { is_p2p_allowed_country } = React.useContext(DerivStore)
     return (
```
Comment (niloo-fs) [inline src/pages/partners/payment-agent/_faq-data.tsx:9]:
```suggestion
const StyledHeader = styled(Header)`
    font-size: 1.6rem;
    
    @media ${device.tabletL} {
        font-size: 12px;
    }
`

    return (
```

File: src/pages/partners/payment-agent/_faq-data.tsx (hunk 1 above)
Comment (meenakshi-deriv) [reply src/pages/partners/payment-agent/_faq-data.tsx:11]: Hi @konstantinosG-deriv , as per the design the size is 14px. So no need to add this prop.

**Review by niloo-fs — APPROVED** [review]

**Review by jim-deriv — APPROVED** [review]

**Review by niloo-fs — APPROVED** [review]

**Pull request merged**